service/src/main/java/uk/gov/hmcts/reform/fpl/exceptions/removeorder/RemovableOrderActionNotFoundException.java

```diff
@@ -1,13 +1,11 @@
 package uk.gov.hmcts.reform.fpl.exceptions.removeorder;
 
-import java.util.UUID;
-
 public class RemovableOrderActionNotFoundException extends IllegalStateException {
     public RemovableOrderActionNotFoundException() {
         super("Removable order action not found");
     }
 
-    public RemovableOrderActionNotFoundException(UUID orderId) {
-        super(String.format("Removable order action not found for order %s", orderId));
+    public RemovableOrderActionNotFoundException(String orderRemovalAction) {
```

Comment:
I think as it was before was the correct one?

Reply:
Should probably be something more along the lines of `Removable order action for order of type x` we care about the type of the order not the order action